testsuite/testsuite-maven-plugin/src/test/java/org/wildfly/swarm/maven/plugin/MavenPluginTest.java

```diff
@@ -267,7 +267,7 @@ public void doBuildUberjarAndRunTests() throws IOException, VerificationExceptio
             assertThat(log).contains("Custom main() usage is intended to be deprecated in a future release");
             // 1st warning for wildfly-swarm:package
             // 2nd warning possibly for wildfly-swarm:start for tests
-            assertThat(count).as("There should only be 1 or 2 warnings").isIn(1, 2);
+            assertThat(count).as("There should only be 1 to 3 warnings").isIn(1, 2, 3);
```

Comment:
Could you please add a comment that the 3rd warning might come from `wildfly-swarm:stop`? Thanks!

Reply:
I did :)